src/webextension/manage/components/homepage.js

```diff
@@ -3,49 +3,32 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import { withLocalization } from "fluent-react";
+import { Localized } from "fluent-react";
 import PropTypes from "prop-types";
 import React from "react";
 
 import styles from "./homepage.css";
-import lockie from "./lockie.txt";
-
-function printableLength(s) {
-  return s.replace(/(\u2068|\u2069)/g, "").length;
-}
-
-function speechBubble(text) {
-  const lines = text.split("\n");
-  const innerWidth = Math.max(...lines.map(printableLength), 1);
-
-  const result = [
-    " " + "_".repeat(innerWidth + 4),
-    "/" + " ".repeat(innerWidth + 4) + "\\",
-    ...lines.map((i) => {
-      return "|  " + i + " ".repeat(innerWidth - printableLength(i)) + "  |";
-    }),
-    "\\__   " + "_".repeat(innerWidth - 1) + "/",
-    "   \\ /",
-    "    v",
-  ];
-  return result.join("\n");
-}
 
 function Homepage({count, getString}) {
+  const imgSrc = browser.extension.getURL("/images/lockie_v2.svg");
+
   let key;
   if (count === 0) {
-    key = "homepage-no-passwords";
-  } else if (count < 10) {
-    key = "homepage-under-10-passwords";
-  } else if (count < 50) {
-    key = "homepage-under-50-passwords";
+    key = "homepage-no-passwords-title";
   } else {
-    key = "homepage-over-50-passwords";
+    key = "homepage-has-passwords-title";
   }
 
   return (
-    <pre className={styles.homepage}>{
-      speechBubble(getString(key, {count})) + "\n" + lockie
-    }</pre>
+    <article className={styles.homepage}>
+      <img src={imgSrc} alt=""/>
+        <h1>{
+          getString(key, {count})
```

Comment:
Let's get rid of `getString()` and use `<Localized>` instead. I should have done that earlier, but the speech bubble code was all temporary, and `getString()` was the simplest way to do it at the time...